test/selenium/print_test.js

```diff
@@ -0,0 +1,14 @@
+// 03 Print test using browserstack
+
+var webdriver = require('browserstack-webdriver');
+
+var runTest = function(cap, driver, target) {
+  // Click on "Drucken"
+  driver.findElement(webdriver.By.xpath("//a[@id='printHeading']")).click();
+  // Wait 1 sec for the menu to pull down
+  driver.sleep(1000);
```

Comment:
Is the class collapsed defined after the animation? It won't help if not.

I know sleep is bad.